lib/server/hqqresolver.py

```diff
@@ -8,17 +8,19 @@
 # *  http://www.gnu.org/copyleft/gpl.html
 # *
 # *
-# *  based on https://gitorious.org/iptv-pl-dla-openpli/ urlresolver
+# *  original based on https://gitorious.org/iptv-pl-dla-openpli/ urlresolver
+# *  update based on https://github.com/LordVenom/
 # */
 from StringIO import StringIO
 import json
-import re
 import util
+import re
 import base64
 import urllib
+import urllib2
 
 __name__ = 'hqq'
-
+UA = 'Mozilla/6.0 (Windows; U; Windows NT 5.1; en-GB; rv:1.9.0.5) Gecko/2008092417 Firefox/3.0.3'
```

Comment:
thank you for your relentless review - hopefully all suggestions are implemented